### PR TITLE
Pioneer MAB pre-cruise deploy csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
pre-cruise deploy csvs:
CP10CNSM
CP13EAPM
CP13NOPM

Refer to config sheets in Pioneer 20 Deployment folder in Vault, and attached marine infrastructure and refdes excel file.

[1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-01-24_Ver_2-04.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/14421202/1100-00006_Marine_Infrastructure_Vocabulary_and_Reference_Designators_OOI_2024-01-24_Ver_2-04.xlsx)

oh...and lat/lon taken from 3210-00007 & table attached...

[MAB_latitudes_longitudes.docx](https://github.com/WHOIGit/ooicgsn-asset-management/files/14422098/MAB_latitudes_longitudes.docx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Pioneer 20 AM Updates](https://trello.com/c/22LKGqi6/232-pioneer-20-am-updates)

File: deployment/CP10CNSM_Deploy.csv [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+AR82,,,,CP10CNSM-MFD36-04-VELPTA000,1,1,2024-02-26T12:00:00,,CGMCP-10CNSM-00001,,CGINS-VELPTA-08544,35.95,-75.125,,29,30,,,
+AR82,,,,CP10CNSM-MFD36-05-PCO2WB000,1,1,2024-02-26T12:00:00,,CGMCP-10CNSM-00001,,CGINS-PCO2WB-C0097,35.95,-75.125,,29,30,,,
+AR82,,,,CP10CNSM-MFD36-06-PHSEND000,1,1,2024-02-26T12:00:00,,CGMCP-10CNSM-00001,,CGINS-PHSEND-P0100,35.95,-75.125,,29,30,,,
+#AR82,,,,CP10CNSM-MFD37-00-DCLENG000,1,1,2024-02-26T12:00:00,,CGMCP-10CNSM-00001,,,35.95,-75.125,,29,30,,,
+AR82,,,,CP10CNSM-MFD37-01-OPTAAD000,1,1,2024-02-26T12:00:00,,CGMCP-10CNSM-00001,,CGINS-OPTAAD-00205,35.95,-75.125,,29,30,,,
+AR82,,,,CP10CNSM-MFD37-02-ADCPTQ000,1,1,2024-02-26T12:00:00,,CGMCP-10CNSM-00001,,CGINS-ADCPTQ-01033,35.95,-75.125,,29,30,,,
+AR82,,,,CP10CNSM-MFD37-03-CTDBPC000,1,1,2024-02-26T12:00:00,,CGMCP-10CNSM-00001,,CGINS-CTDBPC-50111,35.95,-75.125,,29,30,,,
+AR82,,,,CP10CNSM-MFD37-04-DOSTAD000,1,1,2024-02-26T12:00:00,,CGMCP-10CNSM-00001,,CGINS-DOSTAD-00394,35.95,-75.125,,29,30,,,
+AR82,,,,CP10CNSM-MFD37-07-ZPLSCC000,1,1,2024-02-26T12:00:00,,CGMCP-10CNSM-00001,,CGINS-ZPLSCC-55102,35.95,-75.125,,29,30,,,
```
Comment (reedan88) [inline deployment/CP10CNSM_Deploy.csv:55]: Looks good.

File: deployment/CP13NOPM_Deploy.csv [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+AR82,,,,CP13NOPM-RII01-02-ADCPUP021,1,1,2024-02-26T12:00:00,,CGMCP-13NOPM-00001,,CGINS-ADCPUP-00276,36.175,-74.8267,,17,100,,,
+AR82,,,,CP13NOPM-RII01-02-ADCPTG010,1,1,2024-02-26T12:00:00,,CGMCP-13NOPM-00001,,CGINS-ADCPTG-19151,36.175,-74.8267,,82,100,,,
+AR82,,,,CP13NOPM-WFP01-00-WFPENG001,1,1,2024-02-26T12:00:00,,CGMCP-13NOPM-00001,CGVEH-CWFPCP-71401,CGCON-CWFPCP-71401,36.175,-74.8267,,81,100,,,
+AR82,,,,CP13NOPM-WFP01-01-VEL3DK000,1,1,2024-02-26T12:00:00,,CGMCP-13NOPM-00001,CGVEH-CWFPCP-71401,CGINS-VEL3DK-00012,36.175,-74.8267,,81,100,,,
+AR82,,,,CP13NOPM-WFP01-02-DOFSTK000,1,1,2024-02-26T12:00:00,,CGMCP-13NOPM-00001,CGVEH-CWFPCP-71401,CGINS-DOFSTK-00209,36.175,-74.8267,,81,100,,,
+AR82,,,,CP13NOPM-WFP01-03-CTDPFK000,1,1,2024-02-26T12:00:00,,CGMCP-13NOPM-00001,CGVEH-CWFPCP-71401,CGINS-CTDPFK-00093,36.175,-74.8267,,81,100,,,
+AR82,,,,CP13NOPM-WFP01-04-FLORTK000,1,1,2024-02-26T12:00:00,,CGMCP-13NOPM-00001,CGVEH-CWFPCP-71401,CGINS-FLORTK-00830,36.175,-74.8267,,81,100,,,
+AR82,,,,CP13NOPM-WFP01-05-PARADK000,1,1,2024-02-26T12:00:00,,CGMCP-13NOPM-00001,CGVEH-CWFPCP-71401,CGINS-PARADK-20525,36.175,-74.8267,,81,100,,,
```
Comment (reedan88) [inline deployment/CP13NOPM_Deploy.csv:12]: Looks good.

**Review by reedan88 — CHANGES_REQUESTED** [review]
I think the longitude for EAPM is incorrect.

File: deployment/CP13EAPM_Deploy.csv [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+AR82,,,,CP13EAPM-RII01-02-ADCPUP021,1,1,2024-02-26T12:00:00,,CGMCP-13EAPM-00001,,CGINS-ADCPUP-00280,35.95,-74.8457,,17,100,,,
+AR82,,,,CP13EAPM-RII01-02-ADCPTG010,1,1,2024-02-26T12:00:00,,CGMCP-13EAPM-00001,,CGINS-ADCPTG-24543,35.95,-74.8457,,82,100,,,
+AR82,,,,CP13EAPM-WFP01-00-WFPENG001,1,1,2024-02-26T12:00:00,,CGMCP-13EAPM-00001,CGVEH-CWFPCP-99101,CGCON-CWFPCP-99101,35.95,-74.8457,,81,100,,,
+AR82,,,,CP13EAPM-WFP01-01-VEL3DK000,1,1,2024-02-26T12:00:00,,CGMCP-13EAPM-00001,CGVEH-CWFPCP-99101,CGINS-VEL3DK-00015,35.95,-74.8457,,81,100,,,
+AR82,,,,CP13EAPM-WFP01-02-DOFSTK000,1,1,2024-02-26T12:00:00,,CGMCP-13EAPM-00001,CGVEH-CWFPCP-99101,CGINS-DOFSTK-02978,35.95,-74.8457,,81,100,,,
+AR82,,,,CP13EAPM-WFP01-03-CTDPFK000,1,1,2024-02-26T12:00:00,,CGMCP-13EAPM-00001,CGVEH-CWFPCP-99101,CGINS-CTDPFK-00111,35.95,-74.8457,,81,100,,,
+AR82,,,,CP13EAPM-WFP01-04-FLORTK000,1,1,2024-02-26T12:00:00,,CGMCP-13EAPM-00001,CGVEH-CWFPCP-99101,CGINS-FLORTK-01029,35.95,-74.8457,,81,100,,,
+AR82,,,,CP13EAPM-WFP01-05-PARADK000,1,1,2024-02-26T12:00:00,,CGMCP-13EAPM-00001,CGVEH-CWFPCP-99101,CGINS-PARADK-20541,35.95,-74.8457,,81,100,,,
```
Comment (reedan88) [inline deployment/CP13EAPM_Deploy.csv:12]: Looks good now!

**Review by reedan88 — APPROVED** [review]
EAPM longitude fixed. Looks good!

File: deployment/CP10CNSM_Deploy.csv (hunk 1 above)
Comment (cdobs) [inline deployment/CP10CNSM_Deploy.csv:55]: looks good

File: deployment/CP12CNSW_Deploy.csv [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+#AR82,,,,CP12CNSW-SBS01-05-MOPAK0000,1,1,2024-03-01T00:00:00,,CGMCP-12CNSW-00001,,,35.9500,-75.125,,0,30,,,
+AR82,,,,CP12CNSW-SBI01-02-CTDMOS011,1,1,2024-03-01T00:00:00,,CGMCP-12CNSW-00001,,CGINS-CTDMOS-11641,35.9500,-75.125,,1,30,,,
+AR82,,,,CP12CNSW-SBS01-03-VELPTA000,1,1,2024-03-01T00:00:00,,CGMCP-12CNSW-00001,,CGINS-VELPTA-11402,35.9500,-75.125,,1,30,,,
+#AR82,,,,CP12CNSW-WFP01-00-WFPENG001,1,1,2024-03-01T00:00:00,,CGMCP-12CNSW-00001,,,35.9500,-75.125,,24,30,,,
+#AR82,,,,CP12CNSW-WFP01-01-CTDPFP000,1,1,2024-03-01T00:00:00,,CGMCP-12CNSW-00001,,,35.9500,-75.125,,24,30,,,
+#AR82,,,,CP12CNSW-WFP01-02-DOFSTP000,1,1,2024-03-01T00:00:00,,CGMCP-12CNSW-00001,,,35.9500,-75.125,,24,30,,,
+#AR82,,,,CP12CNSW-WFP01-03-FLORTP000,1,1,2024-03-01T00:00:00,,CGMCP-12CNSW-00001,,,35.9500,-75.125,,24,30,,,
+AR82,,,,CP12CNSW-MFI01-02-CTDMOS031,1,1,2024-03-01T00:00:00,,CGMCP-12CNSW-00001,,CGINS-CTDMOS-11648,35.9500,-75.125,,30,30,,,
+AR82,,,,CP12CNSW-MFI01-02-ADCPTQ010,1,1,2024-03-01T00:00:00,,CGMCP-12CNSW-00001,,CGINS-ADCPTQ-01035,35.9500,-75.125,,30,30,,,
+AR82,,,,CP12CNSW-MFI01-02-WAVSSQ010,1,1,2024-03-01T00:00:00,,CGMCP-12CNSW-00001,,CGINS-WAVSSQ-01035,35.9500,-75.125,,30,30,,,
```
Comment (cdobs) [inline deployment/CP12CNSW_Deploy.csv:12]: looks good

File: deployment/CP12WESW_Deploy.csv [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+#AR82,,,,CP12WESW-SBS01-05-MOPAK0000,1,1,2024-03-01T00:00:00,,,CGMCP-12WESW-00001,,35.9500,-75.3333,,0,30,,,
+AR82,,,,CP12WESW-SBI01-02-CTDMOS011,1,1,2024-03-01T00:00:00,,,CGMCP-12WESW-00001,CGINS-CTDMOS-12304,35.9500,-75.3333,,1,30,,,
+AR82,,,,CP12WESW-SBS01-03-VELPTA000,1,1,2024-03-01T00:00:00,,,CGMCP-12WESW-00001,CGINS-VELPTA-12224,35.9500,-75.3333,,1,30,,,
+#AR82,,,,CP12WESW-WFP01-00-WFPENG001,1,1,2024-03-01T00:00:00,,,CGMCP-12WESW-00001,,35.9500,-75.3333,,24,30,,,
+#AR82,,,,CP12WESW-WFP01-01-CTDPFP000,1,1,2024-03-01T00:00:00,,,CGMCP-12WESW-00001,,35.9500,-75.3333,,24,30,,,
+#AR82,,,,CP12WESW-WFP01-02-DOFSTP000,1,1,2024-03-01T00:00:00,,,CGMCP-12WESW-00001,,35.9500,-75.3333,,24,30,,,
+#AR82,,,,CP12WESW-WFP01-03-FLORTP000,1,1,2024-03-01T00:00:00,,,CGMCP-12WESW-00001,,35.9500,-75.3333,,24,30,,,
+AR82,,,,CP12WESW-MFI01-02-CTDMOS031,1,1,2024-03-01T00:00:00,,,CGMCP-12WESW-00001,CGINS-CTDMOS-12337,35.9500,-75.3333,,30,30,,,
+AR82,,,,CP12WESW-MFI01-02-ADCPTQ010,1,1,2024-03-01T00:00:00,,,CGMCP-12WESW-00001,CGINS-ADCPTQ-01036,35.9500,-75.3333,,30,30,,,
+AR82,,,,CP12WESW-MFI01-02-WAVSSQ010,1,1,2024-03-01T00:00:00,,,CGMCP-12WESW-00001,CGINS-WAVSSQ-01036,35.9500,-75.3333,,30,30,,,
```
Comment (cdobs) [inline deployment/CP12WESW_Deploy.csv:12]: looks good

File: deployment/CP13EAPM_Deploy.csv (hunk 3 above)
Comment (cdobs) [inline deployment/CP13EAPM_Deploy.csv:12]: looks good

File: deployment/CP13NOPM_Deploy.csv (hunk 2 above)
Comment (cdobs) [inline deployment/CP13NOPM_Deploy.csv:12]: looks good

**Review by cdobs — APPROVED** [review]

File: deployment/CP12CNSW_Deploy.csv (hunk 4 above)
Comment (reedan88) [reply deployment/CP12CNSW_Deploy.csv:12]: Looks good.

File: deployment/CP12WESW_Deploy.csv (hunk 5 above)
Comment (reedan88) [reply deployment/CP12WESW_Deploy.csv:12]: Looks good.

**Review by reedan88 — APPROVED** [review]
CP12CNSW and CP12WESW look good.

**Pull request merged**